src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -72,6 +72,14 @@ public Startup(IConfiguration configuration, IHostingEnvironment hostingEnvironm
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddLogging(builder =>
+            {
+                builder.AddApplicationInsights(Configuration.GetSection("AppInsights").GetValue<string>("InstrumentationKey"));
+                builder.AddFilter<ApplicationInsightsLoggerProvider>("", HostingEnvironment.IsDevelopment() ? LogLevel.Debug : LogLevel.Information);
+                builder.AddFilter<ApplicationInsightsLoggerProvider>("Microsoft", LogLevel.Information);
```

Comment:
Logging filtering can be done programatically or in appsettings.json (we also have appsettings.Development.json as an override for the Development profile) so I'm not sure why we are setting log levels in code and in the json. Here we have category `"Microsoft"` set in the code but not in the json so somebody expecting to find the log levels from just the json will miss it unless they know to check here too and evaluate the overall combined settings. It's confusing so can you remove it please?

Reply:
This is being looked at as part of wider working around tidying up our logging, this is simple setting things more consistiently